### PR TITLE
Feature/account screen

### LINKED ISSUE #4: Add create account screen design
# REQUIREMENTS
For registration, users should provide names and passwords.
Relevant information for filling out the form must be shown.
A button for moving to the next screen must also be included, though no functionality is required yet.

# DESIGN
2 2 Registro (Paso 2)

# Acceptance Criteria
- [ ] Implementation must match provided design

### PR DESCRIPTION
**Pull request by OwenRamire**
# Account Screen layout
Closes #4
- Create the account screen layout
- Create the new navigation to arrive at the Account screen
- Use the `secureTextEntry` prop into the TextInput components where the users will write their passwords

  ## Screenshots 
navigationIntoAccount

Screenshot_1600196030

Screenshot_1600196017

### CODE REVIEW

File: src/screens/RegisterAccount.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+              </Text>
+            </View>
+            <TextInput style={styles.textInput} secureTextEntry={true} />
+          </View>
+          <TouchableOpacity
+            style={styles.nextBtn}
+            onPress={() => {
+              console.log('Hello world');
+            }}>
+            <Text style={styles.nextBtnText}>Siguiente</Text>
```
Comment (LeyaimMagma) [inline src/screens/RegisterAccount.js:72]: could you create a file with the constant for all the text?
like export const NEXT_MESSAGE_ES = 'Siguiente';

File: src/screens/RegisterAccount.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+          </View>
+          <View style={styles.inputsContainer}>
+            <View style={styles.textContainer}>
+              <Text style={styles.titleText}>Escribe de nuevo la clave</Text>
+              <Text style={[styles.titleText, styles.requiredText]}>
+                Requerido
+              </Text>
+            </View>
+            <TextInput style={styles.textInput} secureTextEntry={true} />
+          </View>
```
Comment (LeyaimMagma) [inline src/screens/RegisterAccount.js:66]: Could you refactor this? try using a custom hook or a component

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: src/screens/RegisterAccount.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <Text style={styles.title}>Crea tu cuenta</Text>
+        <View style={styles.formContainer}>
+          <EmailTxtInput title="Correo electrónico" isRequired="Requerido" />
+          <PassTxtInput title="Clave" isRequired="Requerido" />
+          <PasswordRequirements />
+          <PassTxtInput
+            title="Escribe de nuevo la clave"
```
Comment (LeyaimMagma) [inline src/screens/RegisterAccount.js:26]: can you add this text on Constants.js?

File: src/screens/RegisterAccount.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            isRequired="Requerido"
```
Comment (LeyaimMagma) [inline src/screens/RegisterAccount.js:27]: can you add this text on Constants.js?
IS_REQUIRED = "Requerido"

File: src/components/PasswordRequirements.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          <Text style={styles.dash}>-</Text> contener por lo menos 1 letra
+          minúscula
+        </Text>
+        <Text style={styles.textInfo}>
+          <Text style={styles.dash}>-</Text> contener por lo menos 1 letra
+          mayúscula
+        </Text>
+        <Text style={styles.textInfo}>
+          <Text style={styles.dash}>-</Text> contener por lo menos 1 dígito
+        </Text>
```
Comment (LeyaimMagma) [inline src/components/PasswordRequirements.js:22]: add these texts on Constants.js?

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by LeyaimMagma — DISMISSED** [review]

**Review by xErikPsx — APPROVED** [review]
Well changed!

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**